README.adoc

```diff
@@ -106,33 +100,15 @@ Adding this dependency allows you to use the fault tolerance policies in your mi
 You can also find the `mpFaultTolerance-1.0` feature in your `src/main/liberty/config/server.xml` server configuration,
 which turns on MicroProfile Fault Tolerance capabilities in Open Liberty.
 
-To easily work through this guide, the two provided microservices are set up to run
-on the same server. To simulate the availability of the services and then to enable fault tolerance,
-dynamic configuration with MicroProfile Configuration is used so that you can easily take one service
-or the other down for maintenance. If you want to learn more about setting up dynamic configuration,
-see https://openliberty.io/guides/microprofile-config.html[Configuring microservices].
-
-The following two steps set up the dynamic configuration on the `system` service and its client.
-You can move on to the next section, which adds the fallback mechanism on the `inventory` service.
-
-First, the `src/main/java/io/openliberty/guides/system/SystemResource.java` class has
-the `inMaintenance()` condition, which determines that the system properties are returned only if you
-set the `io_openliberty_guides_system_inMaintenance` configuration property
-to `false` in the `resources/CustomConfigSource.json` file.
-Otherwise, the service returns a `Status.SERVICE_UNAVAILABLE` message, which makes it unavailable.
+To demonstrate the `@Fallback` mechanism, the microservice simulates the `system` service is in maintenance.
+The `system` service is injected with a dynamic configuration property by using MicroProfile Configuration.
+The configuration enables you to take it down or bring it up easily. By changing this property,
+the `system` service can be taken down quickly, simulating a failure that results in an exception,
+which triggers the `@Fallback` mechanism.
```

Comment:
We've strung a few phrases together here. I'd suggest making the last phrase its own sentence: "...simulating a failure that results in an exception. The exception then triggers the..."